Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:12
+
+WORKDIR /app
+COPY . /app
+
+RUN npm install
+RUN npm run build
```

Comment:
I like the idea of using docker here for a consistent local dev run.  However, If we're going to do this, I would prefer for there to be just one way and integrate it into ./interactive.sh (which is in the contribute.md).

Perhaps we can pull this out into a separate PR and not hold up the needed local ref changes below?

Reply:
sure. I have no problem pulling out the docker stuff. I just needed it as I don't like polluting my machine with node :)

in my case I was doing ./docker.build.sh then ./docker.dev.sh and then running all the commands, including interactive in the container.  your call though. if you want it out, lets remove it.

Reply:
yeah, I love the idea of this but let's take in separate PR and merge with interactive.sh